docs/articles_en/get_started/installing-openvino-overview/installing-openvino-linux-header/installing-openvino-zypper.rst

```diff
@@ -0,0 +1,205 @@
+.. {#openvino_docs_install_guides_installing_openvino_zypper}
+
+Install OpenVINO™ Runtime on Linux From ZYPPER Repository
+========================================================
+
+
+.. meta::
+   :description: Learn how to install OpenVINO™ Runtime on Linux operating
+                 system, using the ZYPPER repository.
+
+.. note::
+
+   Note that the ZYPPER distribution:
+
+   * offers both C/C++ and Python APIs
+   * does not offer support for NPU inference
+   * is dedicated to Linux users only
+   * additionally includes code samples
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | :doc:`System Requirements Page <system_requirements>`
+
+      .. note::
+
+         OpenVINO RPM packages are compatible with and can be run on the following operating systems:
+
+         - RHEL 8.2 and higher
+         - Amazon Linux 2022 and 2023
+         - Rocky Linux 8.7, 8.8 and 9.2
+         - Alma Linux 8.7, 8.8 and 9.2
+         - Oracle Linux 8.7, 8.8 and 9.2
+         - Fedora 29 and higher up to 40
+         - OpenEuler 20.03 and 22.03
+         - Anolis OS 8.6 and 8.8
+         - CentOS Stream 8 and 9
+         - openSUSE 15.4 and openSUSE 15.5
+
+   .. tab-item:: Processor Notes
+      :sync: processor-notes
+
+      | To see if your processor includes the integrated graphics technology and supports iGPU inference, refer to:
+      | `Product Specifications <https://ark.intel.com/>`__
+
+   .. tab-item:: Software
+      :sync: software
+
+      * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+      * GCC 8.2.0
+      * `Python 3.8 - 3.11, 64-bit <https://www.python.org/downloads/>`_
+
+
+Install OpenVINO Runtime
+########################
+
+Step 1: Set Up the Repository
++++++++++++++++++++++++++++++
+
+
+1. Create a ZYPPER repository file with the command below:
+
+   .. code-block:: sh
+
+      sudo zypper addrepo https://download.opensuse.org/repositories/home:cabelo:intel/15.5/home:cabelo:intel.repo
```

Comment:
am I right that this command is not needed when OpenVINO will be accepted to official openSUSE?

Reply:
> am I right that this command is not needed when OpenVINO will be accepted to official openSUSE?

correct, this command is not needed when OpenVINO will be  accepted to official openSUSE repository (in progress...)